core/modules/replica/tools/qserv-replica-job-purge.cc

```diff
@@ -0,0 +1,147 @@
+/*
+ * LSST Data Management System
+ * Copyright 2018 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+/// qserv-replica-job-purge.cc is a single job Controller application
+/// which is meant to run the corresponding job.
+
+// System headers
+#include <atomic>
+#include <iostream>
+#include <stdexcept>
+#include <string>
+
+// Qserv headers
+#include "proto/replication.pb.h"
+#include "replica/Controller.h"
+#include "replica/ReplicaInfo.h"
+#include "replica/PurgeJob.h"
+#include "replica/ServiceProvider.h"
+#include "util/BlockPost.h"
+#include "util/CmdLineParser.h"
+
+using namespace lsst::qserv;
+
+namespace {
+
+// Command line parameters
+
+std::string  databaseFamily;
+std::string  configUrl;
+unsigned int numReplicas;
+bool         progressReport;
+bool         errorReport;
+bool         chunkLocksReport;
+
+/// Run the test
```

Comment:
I'm confused now; are these integration tests? If so I think the name of the file should start with "test"?